RecoLocalCalo/CaloTowersCreator/src/CaloTowersCreationAlgo.cc

```diff
@@ -265,7 +265,7 @@ void CaloTowersCreationAlgo::setGeometry(const CaloTowerTopology* cttopo, const
     std::vector<int> tower28depths;
     int ndepths, startdepth;
     theHcalTopology->getDepthSegmentation(theHcalTopology->lastHERing()-1,tower28depths);
-    theHcalTopology->depthBinInformation(HcalEndcap,theHcalTopology->lastHERing()-1,ndepths,startdepth);
+    theHcalTopology->depthBinInformation(HcalEndcap,theHcalTopology->lastHERing()-1,1,1,ndepths,startdepth);
```

Comment:
I did a partial work - just make the code compile. It needs further work. Should I take care of this. I thought it is better the Plan-1 version is to be integrated and then we clean up CaloTower, Validation and DQM code needed for this.